src/Reducers/requestAssetsReducers.js

```diff
@@ -76,6 +76,23 @@ const requestAssetsReducers = (state = initialState.requestAssets, action) => {
         openModal: !state.openModal
       };
     }
+    case types.MAKE_REQUESTS_FETCH: {
+      return { ...state, isLoading: false };
```

Comment:
I think naming makes a little confusing. Instead of `isLoading` it should be `isWaiting`
Cuz Im waiting my response to come back based on the response I am start showing snackbar. That indicates snackbar appears needs to stay there at least 1500ms

Reply:
I'm still confused about what is going on with this `isWaiting` boolean, and what the behavior is supposed to be. I was concerned with the actual values as opposed to the naming of the variable. maybe we could talk more outside github?

Reply:
I will dm u

Reply:
cool! Thanks for making those changes 👍